internal/cmd/fga.go

```diff
@@ -65,6 +66,15 @@ func init() {
 	queryCmd.Flags().String("order", "", "order in which a list of results should be returned (asc or desc)")
 	fgaCmd.AddCommand(queryCmd)
 
+	// schema
+	convertSchemaCMD.Flags().String("to", "json", "output to (schema or json)")
+	convertSchemaCMD.Flags().String("output-format", "pretty", "output format (pretty or raw). use raw for machine-readable output or writing to a file")
+	schemaCmd.AddCommand(convertSchemaCMD)
+	applySchemaCmd.Flags().BoolP("verbose", "v", false, "print extra details about the request")
+	applySchemaCmd.Flags().Bool("fail-on-warnings", false, "fail if there are warnings")
```

Comment:
Thoughts on naming this flag `--strict`?

Reply:
that's fine with me